src/async_impl/client.rs

```diff
@@ -129,20 +213,34 @@ impl ClientBuilder {
         })
     }
 
+    /// Use native TLS backend.
+    #[cfg(feature = "default-tls")]
+    pub fn use_default_tls(mut self, tls: Option<TlsConnectorBuilder>) -> ClientBuilder {
+        self.config.tls = TLSBackend::Default(tls);
+        self
+    }
+
+    /// Use rustls TLS backend.
+    #[cfg(feature = "rustls-tls")]
+    pub fn use_rustls_tls(mut self, tls: Option<rustls::ClientConfig>) -> ClientBuilder {
```

Comment:
What do you think of the just `use_rustls` for the name?

Reply:
For `default-tls`, do we named it `use_default`?

Reply:
No, I sort of felt `rustls_tls` is repetitive, but I don't feel strongly.